changelog.R

```diff
@@ -266,8 +278,7 @@ chglog <- tribble(
 )
 
 
-output$changelog <- renderHtmlTableWidget({ 
-  chglog %>% 
-    addHtmlTableStyle(align="l") %>% 
-    htmlTableWidget(rnames=FALSE) 
-})
+output$changelog <- renderDataTable({
```

Comment:
outputs are kept in server.R. While this does work, I'm thinking we should keep it consistent and move it to server.R